lib/bulkPrefixer.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
This enables [strict mode](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode) which prevents some weird behavior from older versions of javascript